BitmarkRegistry/Services/AssetFileService.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AssetFileService.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/1/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import BitmarkSDK
+
+class AssetFileService {
+
+  // MARK: Properties
+  let owner: Account
+  let assetId: String
+
+  lazy var downloadedFolderURL: URL = {
+    let directoryURL = URL(
+      fileURLWithPath: owner.getAccountNumber() + "/assets/" + assetId + "/downloaded",
+      relativeTo: FileManager.sharedDirectoryURL
+    )
+    try? FileManager.default.createDirectory(at: directoryURL, withIntermediateDirectories: true)
```

Comment:
We need to add TODO here to handle error.
Later on I'll add Sentry for error reporting.

Reply:
`try?` is on purpose (following the tutorial video)
this code line will cover: create directory if the directory has not existed yet; if not, nothing happens

Reply:
Ok.